crates/pathfinder/src/bin/pathfinder/main.rs

```diff
@@ -484,19 +488,33 @@ mod pathfinder_context {
                     database: data_directory.join("mainnet.sqlite"),
                     l1_core_address: H160::from(core_addr::MAINNET),
                 },
-                NetworkConfig::Testnet => Self {
-                    network: Chain::Testnet,
-                    network_id: ChainId::TESTNET,
-                    gateway: GatewayClient::testnet(),
+                NetworkConfig::GoerliTestnet => Self {
+                    network: Chain::GoerliTestnet,
+                    network_id: ChainId::GOERLI_TESTNET,
+                    gateway: GatewayClient::goerli_testnet(),
                     database: data_directory.join("goerli.sqlite"),
-                    l1_core_address: H160::from(core_addr::TESTNET),
+                    l1_core_address: H160::from(core_addr::GOERLI_TESTNET),
                 },
-                NetworkConfig::Integration => Self {
-                    network: Chain::Integration,
-                    network_id: ChainId::INTEGRATION,
-                    gateway: GatewayClient::integration(),
+                NetworkConfig::GoerliIntegration => Self {
+                    network: Chain::GoerliIntegration,
+                    network_id: ChainId::GOERLI_INTEGRATION,
+                    gateway: GatewayClient::goerli_integration(),
                     database: data_directory.join("integration.sqlite"),
-                    l1_core_address: H160::from(core_addr::INTEGRATION),
+                    l1_core_address: H160::from(core_addr::GOERLI_INTEGRATION),
+                },
+                NetworkConfig::SepoliaTestnet => Self {
+                    network: Chain::SepoliaTestnet,
+                    network_id: ChainId::SEPOLIA_TESTNET,
+                    gateway: GatewayClient::sepolia_testnet(),
+                    database: data_directory.join("sepolia.sqlite"),
```

Comment:
Its too late to change the older ones, but let's be more specific now.
```suggestion
                    database: data_directory.join("testnet-sepolia.sqlite"),
```